### ISSUE TITLE
Improvements to workspace switch process

### ISSUE THREAD

**Issue by michaelo** [gitspeak/community#49]
Observation 1)
As far as I can see, to switch between workspaces I now have to clik on the top-element on the left menu, then "Add workspace", follow the wizard and then the new workspace is the only one available.

Possible solution may be to maintain a list of n recent workspaces in that same drop-down menu(?).

If there's any mechanism in here already I'd be happy to be educated :)

**Comment by perborgen** [comment]
1) Doesn't the new workspace pop up as a circle, like on the image below (where I have 8 workspaces stacked as circles)

<img src="https://cdn.filestackcontent.com/l9SNHQmQRHRUNSsZhLZF" width="209" title="Screenshot 2019-04-09 at 13.57.45.png">

**Comment by michaelo** [comment]
No, I can't see any such indicators. Could there be any local issues? I see that the workspace does not persist when I reload the application either (reverts back to "michaelo").

![image.png](https://cdn.filestackcontent.com/CvEwRK8VR1elam2pRaRA)

**Comment by perborgen** [comment]
Oh, ok, that seems like a windows bug! I'll check it out right now.

**Comment by perborgen** [comment]
It's actually because you're not actually a member of the gitspeak_ _organization on GitHub. It should work if you're adding an org which you're a member of on GitHub. Could you give that a shot?

**Comment by michaelo** [comment]
Apologies for the late reply. 

That makes perfect sense. I came across this now only as I tried to report issues to GitSpeak and currently does not have any other organizations to join to test as of now. I'll follow up once I've been able to test.

Might anyway be something to look into regarding the user feedback then? As we're now allowed to onboard any workspace, with no feedback of it being an invalid action.

**Comment by perborgen** [comment]
Yes, sorry about that, the community repo is a bit broken now, but it's on our list to fix it. Until then, how about giving feedback via the Slack channel? :)
